instrumentation/instagocb/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/instana/go-sensor/instrumentation/instagocb
```

Comment:
Good point, I think. We can do it 👍🏾

Reply:
It indeed will look weird. This is more like an open question from my side.

Reply:
At the moment I don't have a strong opinion on this, as we have discussed versioning already in the past with the apparent conclusion that each case can be handled in isolation.
I'd say, go ahead the way you think it's best, and we deal with potential issues as they appear.

Reply:
I also think it would be better to use "v2" in the module name for some added clarity. I'll make that change now since everyone seems to agree with it.

Reply:
BTW, if we change the module name to "v2," our release should start from v2.0.0, correct? Otherwise, it might be confusing. Gocb changed their version name to v2 from the v2.0 major release onwards. However, since we are releasing our library for the first time, our version will be 0.1, right? This might seem a bit odd—having "v2" in the module name but the package actually at version 0.1.

@willianpc @sanojsubran

Reply:
Yes, If we are going with v2, we need to keep the version as 1.0.0 if the file and then do a major version upgrade. That's why I mentioned that this will be a slightly weird thing to do.

Reply:
If that's the case, I believe we can stick with not adding "v2" to the module name. We'll proceed as is, and as @willianpc suggested, we can address any issues that arise along the way.